invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/grid.overrides

```diff
@@ -36,6 +36,7 @@
     .row.footer-bottom, .sub-footer {
       background-color: @footerDarkColor;
       box-shadow: 0 2px 5px rgba(0, 0, 0, .2) inset;
+      padding-bottom: 1rem 0 1rem;
```

Comment:
did you mean:
```suggestion
      padding: 1rem 0 1rem;
```
the attribute with specific direction (-bottom, -top, etc) usually takes only one value, unless the notation changed recently and I am not aware, please correct me :)

Reply:
(accepted your change below)

Reply:
what about this one? still has 3 values and I am not sure I understand what is it supposed to do.